content/documentation/variables.json

```diff
@@ -3,5 +3,6 @@
   "thing1": "thing1",
   "thing2": {
     "thing1": "thing2"
-  }
+  },
+  "dataflow-version": "2.1.0.BUILD-SNAPSHOT"
```

Comment:
Set this 2.1.0.M1 or all the `wget` commands will return a 404 because we don't have a BUILD-SNAPSHOT tag.

Reply:
We need the equivalent of this in the old doc `<propertyregex property="github-tag" override="true" input="${github-tag}" regexp=".*SNAPSHOT" replace="master" />`

Reply:
Damien will create a plugin, so will leave it as is.